sdk/storage/azure-storage-common/src/shared_key_policy.cpp

```diff
@@ -16,20 +17,19 @@ namespace Azure { namespace Storage {
     string_to_sign += Azure::Core::Http::HttpMethodToString(request.GetMethod()) + "\n";
 
     const auto& headers = request.GetHeaders();
-    for (std::string headerName :
-         {"Content-Encoding",
-          "Content-Language",
-          "Content-Length",
-          "Content-MD5",
-          "Content-Type",
-          "Date",
-          "If-Modified-Since",
-          "If-Match",
-          "If-None-Match",
-          "If-Unmodified-Since",
-          "Range"})
+    for (std::string headerName : {"Content-Encoding",
+                                   "Content-Language",
+                                   "Content-Length",
+                                   "Content-MD5",
+                                   "Content-Type",
+                                   "Date",
+                                   "If-Modified-Since",
+                                   "If-Match",
+                                   "If-None-Match",
+                                   "If-Unmodified-Since",
+                                   "Range"})
```

Comment:
why did you make these changes?

Reply:
clang-formating

Reply:
this is weird, because I also clang-formatted it before.